### PR TITLE
Fix currentInterface URL for multi-organization

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/views/dynamic.py [hunk 1]
```diff
@@ -62,11 +62,14 @@ def _fulltextsearch_groups(self):  # pylint: disable=no-self-use
             .all()
         ]
 
-    def _interface(self, interface_config, interface_name, dynamic):
+    def _interface(self, interface_config, interface_name, route_interface_name, dynamic):
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/views/dynamic.py:65]: route_interface_name is not clear for me.
root_interface_name ? or route_name ?

File: geoportal/c2cgeoportal_geoportal/views/dynamic.py [hunk 2]
```diff
@@ -100,7 +103,8 @@ def _interface(self, interface_config, interface_name, dynamic):
 
     @view_config(route_name="dynamic", renderer="json")
     def dynamic(self):
-        interface_name = self.request.get_organization_interface(self.request.params.get("interface"))
+        route_interface_name = self.request.params.get("interface")
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/views/dynamic.py:106]: Seems it might be root_interface_name (interface name before expansion by organization)

**Review by arnaud-morvan — APPROVED** [review]
Minor comment about variable name.

File: geoportal/c2cgeoportal_geoportal/views/dynamic.py (hunk 1 above)
Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/views/dynamic.py]: => original_interface_name with documentation :-)

**Pull request merged**